apps/src/gamelab/animationListModule.js

```diff
@@ -567,19 +567,35 @@ function loadAnimationFromSource(key, callback) {
         // Brute-force recovery step: Remove the animation from our redux state;
         // it looks like it's already gone from the server.
 
-        // Log data about when this scenario occurs
-        firehoseClient.putRecord(
-         'analysis-events',
-            {
-              study: 'animation_no_load',
-              study_group: 'animation_no_load_with_buttons',
-              event: 'animation_not_loaded',
-              data_json: JSON.stringify({'sourceUrl': sourceUrl, 'version': state.propsByKey[key].version,
-                'animationName': state.propsByKey[key].name, 'error': err.message})
-            }
-        );
-
-        dispatch(reportError(`Sorry, we couldn't load animation "${state.propsByKey[key].name}".`, "anim_load", key));
+        if (isOwner()) {
+          // Log data about when this scenario occurs for projects you own
+          firehoseClient.putRecord(
+           'analysis-events',
+              {
+                study: 'animation_no_load',
+                study_group: 'animation_no_load_v2',
+                event: 'animation_not_loaded_owner',
+                data_json: JSON.stringify({'sourceUrl': sourceUrl, 'version': state.propsByKey[key].version,
+                  'animationName': state.propsByKey[key].name, 'error': err.message})
+              }
+          );
+
+          // Display error dialog
+          dispatch(reportError(`Sorry, we couldn't load animation "${state.propsByKey[key].name}".`, "anim_load", key));
+        } else {
+          // Log data about when this scenario occurs for project you don't own
+          firehoseClient.putRecord(
+           'analysis-events',
+              {
+                study: 'animation_no_load',
+                study_group: 'animation_no_load_v2',
+                event: 'animation_not_loaded_viewer',
+                data_json: JSON.stringify({'sourceUrl': sourceUrl, 'version': state.propsByKey[key].version,
+                  'animationName': state.propsByKey[key].name, 'error': err.message})
+              }
+          );
+        }
```

Comment:
You've got a lot of duplication here.  I'd collapse this to avoid accidentally diverging these firehose events in the future:
```js
firehoseClient.putRecord(
  'analysis-events',
     {
       study: 'animation_no_load',
       study_group: 'animation_no_load_v2',
       event: isOwner() ? 'animation_not_loaded_owner' : 'animation_not_loaded_viewer',
       data_json: JSON.stringify({'sourceUrl': sourceUrl, 'version': state.propsByKey[key].version,
         'animationName': state.propsByKey[key].name, 'error': err.message})
     }
 );

if (isOwner()) {
   // Display error dialog
   dispatch(reportError(`Sorry, we couldn't load animation "${state.propsByKey[key].name}".`, "anim_load", key));
}
```